src/Workspaces/Core/Portable/Simplification/SimplificationOptions.cs

```diff
@@ -101,8 +101,10 @@ public static class SimplificationOptions
         /// and the level to which those rules should be enforced.
         /// </summary>
         internal static PerLanguageOption<NamingStylePreferences> NamingPreferences { get; } = new PerLanguageOption<NamingStylePreferences>(nameof(SimplificationOptions), nameof(NamingPreferences), defaultValue: NamingStylePreferences.Default,
-            storageLocations: new OptionStorageLocation[]{
+            storageLocations: new OptionStorageLocation[] {
                 new NamingStylePreferenceEditorConfigStorageLocation(),
-                new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.NamingPreferences")});
+                new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.NamingPreferences[5]"),
```

Comment:
I used 5 instead of 2 after NamingStylePreferences.s_serializationVersion, which is 5.